python/tk_framework_alias/server/socketio/api_request.py

```diff
@@ -161,24 +178,17 @@ def validate(self, request_name):
 
         return request_name == "batch_requests"
 
-    def execute(self, request_name):
-        """
-        Execute the Alias API request to execute the api function.
-
-        :param request_name: The api request name.
-        :type request_name: str
-
-        :return: The return value api function.
-        :rtype: any
-        """
-
-        self.validate(request_name)
+    def get_exec_func(self):
```

Comment:
Follow up from my previous question, I think I understand now that this was replaced by this exec_func to condition the Alias API version. Do you think it's possible to leave the `execute` function and add some decorator that handles the logic from the base class?

Reply:
We could make it a decorator, but I don't think it adds any benefit, and to me makes it harder to follow the code. So for those reasons, I would leave it as is.